### ISSUE TITLE
An autostart option would be great

### ISSUE THREAD

**Issue by mfat** [staf621/ki4a#2]

**Comment by staf621** [comment]
Sure, how would you implement it?
Like an autostart when connecting to a specific network?

**Comment by mfat** [comment]
Yes. Start at boot and start when a specific network is joined. Thanks much.
I also need another feature which is unlikely to be added, but here it is just in case: I want to be able to share the VPN connection over WiFi. (Using wifi tethering)

**Comment by staf621** [comment]
That functionality is already on the app, but it's a root only feature.
Using iptables mode you can share your SSH connection over wifi, bluetooth or usb

**Comment by mfat** [comment]
Thanks. Worked like a charm.

**Comment by ghost** [comment]
As an FYI for anyone looking to accomplish this without root, I'm using Tasker with the AutoInput plugin. Have Tasker launch ki4a on boot, have AutoInput click the connect button in ki4a, then go home to place ki4a into the background. Works great.
